module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,11 @@
+#include "sort.hpp"
+
+#include <functional>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> mail) {
+    mail.sort(std::greater<std::string>());
```

Comment:
it's not the way it should be sorted. Add more test cases and you will see it :)
You should fix this implementation. It doesn't use size actually.